appveyor.yml

```diff
@@ -81,6 +81,7 @@ test_script:
   - "cd empty_folder"
   - "SET JOBLIB_MULTIPROCESSING=0"
   - "python -c \"import nose; nose.main()\" -s pystan.tests.test_basic:TestNormal"
+  - "python -c \"import nose; nose.main()\" pystan.tests.test_multinormal_compilation:TestMultiNormalCompilation"
```

Comment:
Perhaps we could just add both tests pystan.tests.test_basic:TestNormal and pystan.tests.test_multinormal_compilation:TestMultiNormalCompilation? Or is there a reason to drop the first one.